src/test/java/guitests/BoardTests.java

```diff
@@ -0,0 +1,64 @@
+package guitests;
+
+
+import org.junit.Before;
+import org.junit.Test;
+import prefs.Preferences;
+import ui.TestController;
+import ui.UI;
+import ui.issuepanel.PanelControl;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.loadui.testfx.Assertions.assertNodeExists;
+import static org.loadui.testfx.controls.Commons.hasText;
+
+public class BoardTests extends UITest {
+    PanelControl panelControl;
+    @Before
+    public void setup() {
+        UI ui = TestController.getUI();
+        panelControl = ui.getPanelControl();
+
+        Preferences testPref = UI.prefs;
+
+        List<String> boardNames = testPref.getAllBoardNames();
+        boardNames.stream().forEach(testPref::removeBoard);
+    }
+
+    @Test
+    public void duplicateNameTest() {
+        // Save the current board
+        clickMenu("Boards", "Save as");
+        type("board 1");
+        click("OK");
+        assertEquals(1, panelControl.getPanelCount());
+
+        // Create a new panel, then save with the same name
+        // Expected: Dialog shown to confirm duplicate name
+        clickMenu("Panels", "Create");
+        clickMenu("Boards", "Save as");
+        type("board 1");
+        click("OK");
+        assertNodeExists(hasText("A board by the name 'board 1' already exists."));
```

Comment:
It's better to use `waitUntilNodeAppears` here so that if there is any lag in the window appearing, it doesn't fail immediately. This line causes the Travis to fail right now.